packages/core/package.json

```
@@ -3,9 +3,10 @@
   "version": "2.0.4",
   "description": "Priceline Design System",
   "main": "dist/index.cjs.js",
-  "module": "dist/index.esm.js",
+  "module": "dist/index.js",
+  "sideEffects": false,
   "scripts": {
-    "prepare": "rollup --config",
+    "prepare": "babel src/ --out-dir dist --ignore spec.js,story.js; rollup --config",
```

@bertya, think for consistency to existing chained commands in npm-run scripts, and the fact that there'd be a subtil diff in the `;` and `&&` in the unix implication, I'd prefer to align to use `&&. Proposed the edit.

```suggestion
    "prepare": "babel src/ --out-dir dist --ignore spec.js,story.js && rollup --config",
```